widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.kt

```diff
@@ -1436,10 +1436,17 @@ internal class ChatController(
         viewCallback?.smoothScrollToBottom()
     }
 
-    private fun loadChatHistory() {
+    private fun loadChatHistory(subscribeToMessages: Boolean = false) {
```

Comment:
NIT, renaming a bit `subscribeToMessages` to  `subscribeToNewMessages` might be better for understanding.
But, it's not critical, just my opinion.